php-apache/7.0/usr/local/share/env/default_env_variables

```diff
@@ -12,3 +12,9 @@ export TIDEWAYS_API_KEY=${TIDEWAYS_API_KEY:-}
 export TIDEWAYS_CONNECTION=${TIDEWAYS_CONNECTION:-tcp://tideways:9135}
 
 export WORK_DIRECTORY=${WORK_DIRECTORY:-/app}
+
+export BASIC_AUTH_ENABLED=${BASIC_AUTH_ENABLED:-}
+export BASIC_AUTH_REALM=${BASIC_AUTH_REALM:-Protected System}
+export BASIC_AUTH_FILE=${BASIC_AUTH_FILE:-/etc/apache2/htpasswd}
+export BASIC_AUTH_TYPE=${BASIC_AUTH_TYPE:-Basic}
```

Comment:
That'll leave a lot of changes to names of files and inconsistencies with the nginx images. From a OS perspective it'd prob make sense tho, but would currently miss out `AuthDigestAlgorithm`

A prefix like "AUTH_HTTP_" might make more sense (I'd not recommend "HTTP_AUTH" though due to some languages treating HTTP_ as a request header lookup)